src/integration/pages/homePage/arabic/canonical.test.js

```diff
@@ -8,7 +8,7 @@ import runCanonicalTests from '../canonicalTests';
 import { data as pageData } from '../../../../../data/arabic/homePage/index.json';
 
 describe('Canonical', () => {
-  describe(pageType, () => {
+  describe.only(pageType, () => {
```

Comment:
Reminder to remove this when we're done
```suggestion
  describe(pageType, () => {
```